docs/learn/learn-guides-bounties.md

```diff
@@ -56,6 +56,19 @@ Submitting a bounty proposal will reserve
 
 ## Assign a Curator to a Bounty
 
+:::info
+
+The curator assignment call can only be executed successfully if the original bounty proposal
+funding has been executed and a new spending period has not begun. Proposers need to ensure they
```

Comment:
"has begun" or "has not begun"? The followup statement seems to suggest otherwise.